pom.xml

```diff
@@ -95,28 +99,9 @@
             <version>${io.opentracing.version}</version>
         </dependency>
         <dependency>
-            <groupId>io.grpc</groupId>
```

Comment:
Basically the whole point of all these lines of code across all these repos is to delete these ~20 lines of dependencies. 👍